### ISSUE TITLE
Structure board not positioning placeholders after 3.1.2 upgrade

### ISSUE THREAD

**Issue by marksweb** [divio/django-cms#4314]
Really odd one this is, but I've just pushed an upgrade to 3.1.2 which was fine testing on my machine, but now it's on a dev server all the placeholders are rendering on top of each other in the top left side of the page & I've got a JS error on this like;

```JS
    // start calculating
    this.placeholders.each(function (index, item) {
        item = $(item);
        id = item.data('settings').placeholder_id;
```
Where `item.data('settings')` is Undefined.

Dev server is essentially the same python environment as my machine as it has the same requirements installed so I'm a little confused.
Any ideas?

<img width="679" alt="screen shot 2015-07-26 at 16 54 57" src="https://cloud.githubusercontent.com/assets/1461191/8894668/44d1e4f6-33b7-11e5-95a4-40fdc3c40db3.png">

Actually, I've downgraded to 3.1.1 & it still occurring so don't think it's related to the update.

**Comment by marksweb** [comment]
This morning this has stopped happening completely, so I've not been able to investigate any further.

**Comment by marksweb** [comment]
Ok, this seems very hit & miss. It's happening again, but still the only error comes from item.data('settings') being Undefined.

**Comment by yakky** [comment]
Can't reproduce this. In 3.2 the whole frontend will change a lot in its internals (most of the changes have already landed), with saner behavior and a new option that tiles placeholders instead of trying to map frontend position (which causes so many side effects that it's almost unfixable)

**Comment by marksweb** [comment]
No problem, I didn't think you would be able to @yakky. I raise these odd ones simply in the hope that someone might have come across it.

If I can clear the cache to fix, that's good enough for me. 

Is there an estimated date for 3.2 yet? Don't think I've seen it on any announcements yet.

**Comment by yakky** [comment]
No date announced yet, but September is a sensible goal

**Comment by FinalAngel** [comment]
You can checkout RC3.2, this has been adapted :)

**Comment by rajits** [comment]
Was this issue ever resolved? I'm seeing it in 3.0.12.

**Comment by FinalAngel** [comment]
@rajits I'd recommend updating to latest CMS Version (3.2.x) as we're soon gonna drop support for 3.0.x.
